src/include/ipc4/error_status.h

```diff
@@ -0,0 +1,158 @@
+/* SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Sorry silly question: What is ipc4?

Reply:
currently next ipc version for intel platform